torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -350,12 +349,28 @@ def _functionalized_f_helper(*args):
         disable_above = torch._C._ExcludeDispatchKeyGuard(
             torch._C.DispatchKeySet(torch._C.DispatchKey.Functionalize)
         )
-        with disable_above, FunctionalTensorMode(aot_config.pre_dispatch):
+
+        with disable_above:
```

Comment:
what happened here? it looks like we used to create a new FunctionalTensor mode but we no longer do?

Reply:
Yeah, it seems to not be necessary as this call gets wrapped with [_create_graph](https://github.com/pytorch/pytorch/blob/3eefe962974be9cd70db7717894c010207f9d88b/torch/_functorch/_aot_autograd/dispatch_and_compile_graph.py#L34) which already has a FunctionalMode. 

Although I am breaking some tests.. not sure if this is related